build failure with PAM

```
  CCLD     list_subid_ranges
/usr/bin/ld: ../libsubid/.libs/libsubid.so: undefined reference to `pam_strerror'
/usr/bin/ld: ../libsubid/.libs/libsubid.so: undefined reference to `pam_start'
/usr/bin/ld: ../libsubid/.libs/libsubid.so: undefined reference to `misc_conv'
/usr/bin/ld: ../libsubid/.libs/libsubid.so: undefined reference to `pam_end'
/usr/bin/ld: ../libsubid/.libs/libsubid.so: undefined reference to `pam_chauthtok'
collect2: error: ld returned 1 exit status
```

--- a/libsubid/Makefile.am
+++ b/libsubid/Makefile.am
@@ -16,7 +16,8 @@ MISCLIBS = \
 	$(LIBCRYPT) \
 	$(LIBACL) \
 	$(LIBATTR) \
-	$(LIBTCB)
+	$(LIBTCB) \
+	$(LIBPAM)
 
 libsubid_la_LIBADD = \
 	$(top_srcdir)/lib/libshadow.la \

